assets/iframe.html.tmpl

```diff
@@ -27,6 +27,30 @@
         src="about:blank">
     </iframe>
   <script nonce="{{.Nonce}}">
+    function getAuthTokenWithRetries(maxRetries = 3) {
```

Comment:
@enzowritescode wants to keep these out of the global scope.  See https://github.com/mattermost/mattermost-plugin-msteams-devsecops/pull/24

Reply:
I pushed a change to move `getAuthTokenWithRetries` out of global scope.